blacktie/integration1/README

```diff
@@ -40,9 +40,8 @@ On Windows:
 You MUST use an environment configured with the VC++ development tools, e.g. launch:
 C:\Program Files\Microsoft Visual Studio 9.0\VC\vcvarsall.bat
 
-2. DEPLOY THE BLACKTIE ADMIN SERVICE
-Undeploy any older versions of the blacktie admin service
-Copy the file $BLACKTIE_HOME/blacktie-admin-services/blacktie-admin-services-<VERSION>.ear into $JBOSS_HOME/standalone/deployments
+2. INSTALL THE BLACKTIE SUBSYSTEM AND DEPLOY THE ADMIN SERVICE
+Follow the instructions in https://github.com/jbosstm/narayana/blob/master/blacktie/blacktie/README
```

Comment:
Because the user should be working from a downloaded zip, I think this and similar lines should say:
Follow the instructions in $BLACKTIE_HOME/README

Referencing master would mean that it is possible a user could be using a very old version of BlackTie but trying to follow the instructions in the current github master which could be completely different. Does that make sense?